packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,255 @@
+import { EventProcessor, Hub, Integration, Scope } from '@sentry/types';
+
+interface AWSLambdaContext {
+  getRemainingTimeInMillis: () => number;
+  callbackWaitsForEmptyEventLoop: boolean;
+  awsRequestId: string;
+  functionName: string;
+  functionVersion: string;
+  invokedFunctionArn: string;
+  logGroupName: string;
+  logStreamName: string;
+}
+
+interface Module {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  exports: any;
+  id: string;
+  filename: string;
+  loaded: boolean;
+  parent: Module | null;
+  children: Module[];
+  path: string;
+  paths: string[];
+}
+/**
+ * NodeJS integration
+ *
+ * Provides a mechanism for NodeJS
+ * that raises an exception for handled, unhandled, timeout and similar times of error
+ * and captures the same in Sentry Dashboard
+ */
+export class AWSLambda implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'AWSLambda';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = AWSLambda.id;
+
+  /**
+   * context.
+   */
+  private _awsContext!: AWSLambdaContext;
```

Comment:
This is not true. If user doesnt provide `options.context` it will be `undefined`, so `!` here is not valid

Reply:
Assign the empty object to  _awsContext and added validation.